app/views/images/index.html.erb

```diff
@@ -1,16 +1,15 @@
-<h1>Listing Image URLs</h1>
+<h1>Welcome!</h1>
 
 <%= link_to 'New image', new_image_path %>
 
 <table>
   <tr>
-    <th>Imagel URL</th>
+    <th>Images</th>
   </tr>
 
-  <% @images.each do |image| %>
+  <% @images.reverse_each do |image| %>
```

Comment:
What are your thoughts on sorting by `id` vs `created_at`?

Reply:
In the current implementation, we generate the `@images` array from `Image.all`, which, I assume returns an array in some order (possibly by `id`).  The view renders the images represented in the array in reverse order.

Since the requirement in the issue states, "Newest images appear first," it makes sense to sort on `created_at`, in the event that the returned order and the `created_at` order don't match up (we are currently assuming the orders are the same).

Further, it seems better to provide the view with the array sorted the way we want, rather that it choosing the iteration order.